src/ui/components/Initiatives/InitiativeCreate.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import React from "react";
+import {v4 as uuid} from "uuid";
+import {
+  AutocompleteArrayInput,
+  DateInput,
+  ArrayInput,
+  NumberInput,
+  SimpleForm,
+  TextInput,
+  Create,
+  SimpleFormIterator,
+  BooleanInput,
+} from "react-admin";
+import {
+  getPlainDescFromMd,
+  dateFormatter,
+  dateParser,
+} from "../../initiativeUtils";
+import {CredView} from "../../../analysis/credView";
+
+export const InitiativeCreate = (credView: CredView) => (props: Object) => {
+  const defaultInitiativeValues = {
+    id: uuid(),
+    timestampMs: Date.now(),
+    champions: [],
+    dependencies: [],
+    references: [],
+    contributions: [],
+    weight: {incomplete: 0, complete: 0},
+    completed: false,
+  };
+  return (
+    <Create title="Create New Initiative" {...props}>
+      <SimpleForm initialValues={defaultInitiativeValues}>
```

Comment:
This component is almost exactly the same as the InitiativeEdit component, except this has the defaultInitialValues. Should extract this inner SimpleForm component into a separate `InitiativeForm` component that takes an optional `initialValues` prop and then use that component for both InitiativeEdit and InitiativeCreate. Will make it easier to maintain that way

Reply:
addressed this in #1968